test/CreatorToken.t.sol

```diff
@@ -654,6 +662,26 @@ abstract contract Selling is CreatorTokenTest {
     creatorToken.sell(_tokenId);
     vm.stopPrank();
   }
+
+  function test_RevertIf_MinHoldingTimeNotReached(address _seller) public {
+    // `buyAToken` buys a token and warps to the minimum holding time.
+    buyAToken(_seller);
+    uint256 _tokenId = creatorToken.lastId();
+    // Warp back to before the minimum holding time.
+    vm.warp(block.timestamp - MIN_HOLDING_TIME);
```

Comment:
We should fuzz over the time that has elapsed with an additional parameter that is bounded.

Reply:
Updated in the same commit as above: [77d2ad4](https://github.com/showtime-xyz/creator-tokens/pull/38/commits/77d2ad4397a0e6a3979a7554cb64398d9a505ccf)